lib/benchee/benchmark/scenario.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Benchee.Benchmark.Scenario do
+  defstruct [:job_name, :function, :input_name, :input, :run_times,
+             :run_time_statistics, :memory_usages, :memory_usage_statistics]
+
+  @type t :: %__MODULE__{
+    job_name: binary | nil,
+    function: fun | nil,
+    input_name: binary | nil,
+    input: any | nil,
+    run_times: [non_neg_integer] | nil,
```

Comment:
I think run_times can end up being floats when we run a benchmark multiple times and then divide them by n... so I think it'd be non_neg_number or whatever :)